ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -49,6 +50,235 @@ uint64_t getNewTraceIdx() {
 // The name prefix used for blocks that are branched to when a guard succeeds.
 #define GUARD_SUCCESS_BLOCK_NAME "guardsuccess"
 
+// Describes a "resume point" in an LLVM IR basic block.
+//
+// This is required because in LLVM IR, a call instruction does not terminate a
```

Comment:
Can we make this a bit easier to parse? Maybe something like "Since call instructions are not terminators in LLVM IR, we have to keep track of which call within a block we last processed."

Reply:
I think that throws away too much information.

Asides from the typo, which part is hard to parse, and I'll see what I can do.

Reply:
Fixed in f44ae80727afd814e0c221542e9aa2ddccb838fd